src/bidiServer/bidiServerRunner.ts

```diff
@@ -149,8 +150,37 @@ export class BidiServerRunner {
         }
 
         const plainCommandData = message.utf8Data;
-
         debugRecv(plainCommandData);
+
+        // Try to parse the message to handle some of BiDi commands.
+        let parsedCommandData: {id: number; method: string};
+        try {
+          parsedCommandData = JSON.parse(plainCommandData);
+        } catch (e) {
+          this.#respondWithError(
+            connection,
+            {},
+            'invalid argument',
+            `Cannot parse data as JSON`
```

Comment:
Print the data as well?

Reply:
It is printed [here](https://github.com/GoogleChromeLabs/chromium-bidi/blob/20b8b8fb8ac08298c5cb1d81ff7e6e4cb7ede7a4/src/bidiServer/bidiServerRunner.ts#L153).